larq/layers.py

```diff
@@ -820,58 +837,59 @@ class QuantConv2DTranspose(QuantizerBase, tf.keras.layers.Conv2DTranspose):
+            If set to `None` (default), the output shape is inferred.
+        data_format: A string, one of `channels_last` (default) or `channels_first`. The
+            ordering of the dimensions in the inputs. `channels_last` corresponds to
+            inputs with shape `(batch, height, width, channels)` while `channels_first`
+            corresponds to inputs with shape `(batch, channels, height, width)`. It
+            defaults to the `image_data_format` value found in your Keras config file at
+            `~/.keras/keras.json`. If you never set it, then it will be "channels_last".
+        dilation_rate: an integer or tuple/list of 2 integers, specifying the dilation
+            rate to use for dilated convolution. Can be a single integer to specify the
+            same value for all spatial dimensions. Currently, specifying any
+            `dilation_rate` value != 1 is incompatible with specifying any stride value
+            != 1.
+        activation: Activation function to use. If you don't specify anything,
+            no activation is applied (`a(x) = x`).
+        use_bias: Boolean, whether the layer uses a bias vector.
+        input_quantizer: Quantization function applied to the input of the layer.
+        kernel_quantizer: Quantization function applied to the `kernel` weights matrix.
+        kernel_initializer: Initializer for the `kernel` weights matrix.
+        bias_initializer: Initializer for the bias vector.
+        kernel_regularizer: Regularizer function applied to the `kernel` weights matrix.
+        bias_regularizer: Regularizer function applied to the bias vector.
+        activity_regularizer: Regularizer function applied to
+            the output of the layer (its "activation").
+        kernel_constraint: Constraint function applied to the kernel matrix.
+        bias_constraint: Constraint function applied to the bias vector.
 
     # Input shape
-    4D tensor with shape:
-    `(batch, channels, rows, cols)` if data_format='channels_first'
-    or 4D tensor with shape:
-    `(batch, rows, cols, channels)` if data_format='channels_last'.
+        4D tensor with shape:
+        `(batch, channels, rows, cols)` if data_format='channels_first'
+        or 4D tensor with shape:
+        `(batch, rows, cols, channels)` if data_format='channels_last'.
 
     # Output shape
-    4D tensor with shape:
-    `(batch, filters, new_rows, new_cols)` if data_format='channels_first'
-    or 4D tensor with shape:
-    `(batch, new_rows, new_cols, filters)` if data_format='channels_last'.
-    `rows` and `cols` values might have changed due to padding.
+        4D tensor with shape:
+        `(batch, filters, new_rows, new_cols)` if data_format='channels_first'
+        or 4D tensor with shape:
+        `(batch, new_rows, new_cols, filters)` if data_format='channels_last'.
+        `rows` and `cols` values might have changed due to padding.
 
     # References
     - [A guide to convolution arithmetic for deep
```

Comment:
It looks like this needs another indentaion

Reply:
Oops, fixed last night but hadn't pushed yet.